docs/step-definitions.md

```diff
@@ -279,4 +279,9 @@ These steps are specific to the DVSA MOT application screens.
 * I go to the next tab
 * I close extra tabs
 * I click the accordion section with the id ```..text..```
-* I click the ```..text..``` help link
\ No newline at end of file
+* I click the ```..text..``` help link
+
+### ElasticSearchStepDefinitons
+
+* I query elastic search with ```"..csv file name.."``` as ```"..data key.."```
+* I compare the search results for ```"..result set key.."``` and ```"..result set key.."```
```

Comment:
Does not match method signature.

Also no entry for the new "I set the starting url..."

Reply:
changed the compare step and added the starting url